app/routes/errors.py

```diff
@@ -130,4 +133,17 @@ def fulfilment_request_failed(error):
         blueprint_method, list_item_id=request.view_args["list_item_id"], **request.args
     )
 
-    return _render_error_page(500, template="fulfilment-request", retry_url=retry_url)
+    return _render_error_page(
+        500, template="500-individual-response", retry_url=retry_url
+    )
+
+
+@errors_blueprint.app_errorhandler(ConfirmationEmailFulfilmentRequestFailed)
+def confirmation_email_failed(error):
```

Comment:
To be consistent with the suggested change above:
```suggestion
def confirmation_email_request_failed(error):
```

Reply:
Changed in latest changes.